### PR TITLE
0266 full canvas annotation

### PR DESCRIPTION
**Pull request by robcast**
This PR addresses #266.

### CODE REVIEW

**Comment by glenrobson** [comment]
Direct link here: https://preview.iiif.io/cookbook/0266-full-canvas-annotation/recipe/0266-full-canvas-annotation/

**Comment by glenrobson** [comment]
Code and title to:

https://github.com/IIIF/cookbook-recipes/blob/master/_includes/links.md

and then add link to it in the root index.md

**Comment by glenrobson** [comment]
Comments from the cookbook editors:

Would it be possible to refocus the recipe to be a basic annotation rather than focusing on it being a full canvas annotation? This would mean:

 * Change title from: "Full Canvas Annotation" to "Basic annotation"
 * Change the use case to something like "I want to annotate a canvas with some text either a comment or a tag. In this recipe we annotating the full canvas but see [Simple Commenting Annotation][260] if you would like to target a specific part or region of the canvas. 

Other comments:
 * Remove “​​View in Universal Viewer | View in Tify | View in IIIF Curation Viewer” to only Mirador 
 * Remove Summary, rights and requiredStatement from the manifest as they are not required.
 * Related - as its basic add basic IIIF image, audio + video

**Comment by robcast** [comment]
I integrated your suggestions @glenrobson. I changed the title to "Basic Annotation" but the directory name is still `0266-full-canvas-annotation`. Should that be changed too? (I don't know if that would break anything)

**Comment by glenrobson** [comment]
Its up to you regarding changing the directory name. If you do you will need to also change it in the `links.md` but that should be the only place.

**Comment by glenrobson** [comment]
I've checked the changes and they all look good apart from:

  * Related - as its basic could you add links to the basic IIIF image, basic audio and basic video recipes

If you get chance to make the change then great but I don't think its a major issue so Ill get this ready for the TRC meeting next week.

File: recipe/0266-full-canvas-annotation/index.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+title: Basic Annotation
+id: 266
+layout: recipe
+tags: [tbc]
+summary: "tbc"
+---
+
+## Use Case
+
+I want to annotate a canvas with some text either as a comment or a tag. In this recipe we annotating the full canvas but see [Simple Commenting Annotation][260] if you would like to target a specific part or region of the canvas.
```
Comment (tpendragon) [inline recipe/0266-full-canvas-annotation/index.md:11]: So small, but `we are annotating`

**Comment by glenrobson** [comment]
Hi @robcast now the voting has finished and been approved you can make the changes requested by the TRC.

**Pull request merged**